docs/en/user_guides/finetune.md

```diff
@@ -228,3 +228,61 @@ It's because our training schedule is for a batch size of 128. If using 8 GPUs,
 just use `batch_size=16` config in the base config file for every GPU, and the total batch
 size will be 128. But if using one GPU, you need to change it to 128 manually to
 match the training schedule.
+
+## Evaluate the fine-tuned model on ImageNet variants
+
+It's a common practice to evaluate the ImageNet-(1K, 21K) fine-tuned model the ImageNet-1K validation set. This set
+shares similar data distribution with the training set, but in real world, the inference data is more likely to share
+different data distribution with the training set. To have a full evaluation of model's performance on
+out-of-distribution datasets, research community introduces the ImageNet-variant datasets, which shares different data
+distribution with that of ImageNet-(1K, 21K)., MMClassification supports evaluating the fine-tuned model on
+[ImageNet-A](https://arxiv.org/pdf/1907.07174v4.pdf), [ImageNet-R](https://arxiv.org/pdf/2006.16241v3.pdf),
+[ImageNet-C](https://arxiv.org/pdf/1903.12261v1.pdf), and [ImageNet-S](https://arxiv.org/pdf/2106.03149v3.pdf).
```

Comment:
Better to use a full name since the abbreviation may conflict. And I prefer to provide the link to abstract pages.
```suggestion
[ImageNet-Adversarial (A)](https://arxiv.org/abs/1907.07174), [ImageNet-Rendition (R)](https://arxiv.org/abs/2006.16241),
[ImageNet-Corruption (C)](https://arxiv.org/abs/1903.12261), and [ImageNet-Sketch (S)](https://arxiv.org/abs/1905.13549).
```